{
  "id": "MAL-2024-6259",
  "summary": "Malicious code in zorosnitro (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:44:31Z",
  "published": "2024-06-25T13:44:31Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:44:31Z",
        "import_time": "2024-06-28T02:51:15.344161356Z",
        "versions": [
          "1.0"
        ],
        "id": "RLMA-2024-05063",
        "source": "reversing-labs",
        "sha256": "ca77159a40dbbbab87e2a44926598eab99c60f8dea6f2adfb6b9fa23c8eb81f9"
      }
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "zorosnitro",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/zorosnitro"
      },
      "versions": [
        "1.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/zorosnitro/MAL-2024-6259.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}